everett/manager.py

```diff
@@ -240,6 +241,45 @@ def __repr__(self):
         return '<ListOf(%s)>' % qualname(self.sub_parser)
 
 
+class ExistingFile(object):
+    """A parser for files that are expected to already exist"""
+
+    def __init__(self, mode='', isdir=None):
+        """Check if the file is any subset of readable ('r'), writable ('w') and executable ('x')
+           If 'isdir' is True, also check that it's a directory and not just any kind of file
+           Ignore any other characters in 'mode'
+           Default to 'r' if mode is unspecified, or 'rx' if isdir is True
```

Comment:
Why cram these things into a single string and not break it out into separate arguments? Is this mimicking something else?

Reply:
To clarify, by "these things", I mean the modes to test for--readable, writeable, executable.

Reply:
oh, I was mimicking the `open()` arguments.  I can break them out if you'd rather; I think it's easier to write `parser=ExistingFile('rw')` than `parser=ExistingFile(readable=True, writeable=True)` though.